src/ui/core/zcl_abapgit_gui.clas.abap

```diff
@@ -17,11 +17,6 @@ CLASS zcl_abapgit_gui DEFINITION
         go_back_to_bookmark TYPE i VALUE 6,
         new_page_replacing  TYPE i VALUE 7,
       END OF c_event_state .
-    CONSTANTS:
-      BEGIN OF c_action,
-        go_home TYPE string VALUE 'go_home',
```

Comment:
The idea of defining `go_home` here was to keep the GUI_CORE abstract from the rest of the AG code. Not that this is super critical of course. I'd keep the constant here but refer the VALUE to `zif_abapgit_definitions=>c_action-go_home`